README.md

```diff
@@ -1,3 +1,9 @@
+The changes in this branch are expiremental, and need review (and agreement on the
+changes to the Sphincs+ definition) before pulling.
+It involves using SHA-512 for the H, T functions (rather than SHA-256) in the L3, L5
+parameter sets.  For robust parameter sets, we switch to SHA-512 as well for the masks
+(again, for L3, L5 parameter sets).
+
```

Comment:
`SHA256SUMS` needs to be updated; can be done by using the output of `python3 vectors.py`

Reply:
Done.